articles/connections/social/devkeys.md

```diff
@@ -28,6 +28,6 @@ When using the Auth0 developer keys, there are a few caveats you need to be awar
 
 4. [Federated Logout](/logout#log-out-a-user) does not work. When using the Auth0 developer keys, calling `/v2/logout?federated` will sign the user out of Auth0, but not out of the Social Identity Provider.
 
-5. `prompt=none` won't work on the [/authorize](/api/authentication/reference#social) endpoint. [Auth0.js' `silentAuthentication` method](https://github.com/auth0/auth0.js#silent-authentication) uses `prompt=none` internally, so that won't work either.
```

Comment:
Looks like `renewAuth` is using `prompt=none` internally so I should probably leave the sentence in the doc and update the method name and link. 

[Use prompt=none to retrieve a new token](https://auth0.com/docs/api-auth/tutorials/silent-authentication#use-prompt-none-to-retrieve-a-new-token)